### PR TITLE
[VL] Enhance spill log readability

### PR DESCRIPTION
**Pull request by Yohahaha**

### CODE REVIEW

**Review by FelixYBW — DISMISSED** [review]

File: cpp/velox/compute/WholeStageResultIterator.cc [hunk 1]
```diff
@@ -238,17 +238,21 @@ int64_t WholeStageResultIterator::spillFixedSize(int64_t size) {
       // As of now, non-zero running threads usually happens when:
       // 1. Task A spills task B;
       // 2. Task A trys to grow buffers created by task B, during which spill is requested on task A again;
-      VLOG(2) << logPrefix << "Spill is requested on a task " << task_->taskId()
-              << " that has non-zero running threads, which is not currently supported. Skipping.";
+      LOG(WARNING) << logPrefix << "Spill is requested on a task " << task_->taskId()
+                   << " that has non-zero running threads, which is not currently supported. Skipping.";
```
Comment (zhztheplayer) [inline cpp/velox/compute/WholeStageResultIterator.cc:242]: I think this case is expected but just rare. Can we keep it in INFO? We can lower the verbos info anyway.

File: cpp/velox/compute/WholeStageResultIterator.cc [hunk 2]
```diff
@@ -238,17 +238,21 @@ int64_t WholeStageResultIterator::spillFixedSize(int64_t size) {
...
       return shrunken;
     }
     int64_t remaining = size - shrunken;
-    LOG(INFO) << logPrefix << "Trying to request spill for " << remaining << " bytes...";
-    auto* mm = memoryManager_->getMemoryManager();
+    LOG(INFO) << logPrefix << "Trying to request spill for " << velox::succinctBytes(remaining);
```
Comment (zhztheplayer) [inline cpp/velox/compute/WholeStageResultIterator.cc:246]: nit: With the punctuation mark removed the log could look a bit ugly. E.g., 

```
W20240920 11:38:50.542661 20191 HashBuild.cpp:1052] Trying to request spill for 1MB
```

File: cpp/velox/compute/WholeStageResultIterator.cc (hunk 1 above)
Comment (Yohahaha) [reply cpp/velox/compute/WholeStageResultIterator.cc]: done

File: cpp/velox/compute/WholeStageResultIterator.cc (hunk 2 above)
Comment (Yohahaha) [reply cpp/velox/compute/WholeStageResultIterator.cc]: done

**Review by zhztheplayer — APPROVED** [review]

**Pull request merged**